README.md

```diff
@@ -71,47 +46,44 @@ After you're finished please remove all the comments and instructions!
 <details>
   <summary>Client</summary>
   <ul>
-    <li><a href="https://reactjs.org/">React.js</a></li>
+    <li><a href="https://getbootstrap.com/docs/5.2/getting-started/introduction/">Bootstrap</a></li>
+    <li><a href="https://sass-lang.com/">Sass</a></li>
+    <li><a href="https://www.javascript.com/">JavaScript</a></li>
   </ul>
 </details>
 
 <details>
   <summary>Server</summary>
   <ul>
-    <li><a href="https://expressjs.com/">Express.js</a></li>
+    <li><a href="#">Not Applicable</a></li>
   </ul>
 </details>
 
 <details>
 <summary>Database</summary>
   <ul>
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+    <li><a href="#">Not Applicable</a></li>
   </ul>
 </details>
 
-<!-- Features -->
 
 ### Key Features <a name="key-features"></a>
 
 > Describe between 1-3 key features of the application.
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- **[The speakers section is created dynamically with JavaScript]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
 > Add a link to your deployed project.
 
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
+- [Watch it in action](https://dgcuenca.github.io/nftGallery/)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- I think you need to update your live demo link because your project is rendered correctly when I go live after cloning your repo, but on the deployed link, the content is incomplete. 

⚠️ Kindly note that it is important for your live demo to have ALL updates of your code because that's the link hiring managers or friends will use in viewing your work. 🤗

Reply:
hello @Favourezeugwa  nice to meet you, hope you are well.
The page is deployed with the main branch so when this pull request will be approved then I am gonna merge and all the changes are going to take effect in the deployed link.

Reply:
Okay, that's totally fine @dgcuenca 
Keep rocking! ✈️ 🤝